src/stores/MapOptionsStore.ts

```diff
@@ -193,10 +199,9 @@ const styleOptions: StyleOption[] = [
     kurviger,
     mapillion,
     osmDe,
-    // The original client has these but those options yield cors errors with mapbox yields a cors error
-    // lyrk,
-    // wanderreitkarte,
-    // This works but is extremely slow with mapbox
+    lyrk,
+    wanderreitkarte,
+    // This one is extremely slow with mapbox and openlayers?!
```

Comment:
lyrk and wanderreitkarte work with OpenLayers while they did not for Mapbox. sorbian is still very slow